agents/cdu/fence_cdu.py

```diff
@@ -105,20 +142,17 @@ def main():
     docs["vendorurl"] = "http://www.servertech.com"
     show_docs(options, docs)
 
-    ## Support for --plug [switch]:[plug] notation that was used before
```

Comment:
We shouldn't remove parameters, as that might break setups on upgrade, or people following instructions where it's used.